.gitignore

```diff
@@ -1,6 +1,8 @@
 autosave.math
 .DS_Store
+Vagrantfile
 .vagrant
+install.sh
```

Comment:
I have them in my local for testing on linux at the moment, feel free to remove that if you think its better.  I was trying to completely remove vagrant from the python package to be more like other packages

Reply:
All vagrant stuff is out of .gitignore now.  I almost kept install.sh but figured it was better to remove that file

I created a new repository for all vagrant related items.  I have some plans for this repo, but at the moment it's main function is to say it's where I am planning to put tests at an unspecified time in the future (in case anyone asks if there are tests)

https://github.com/MathInspector/mathinspector-vagrant